cypress/integration/options/totals.cy.js

```diff
@@ -47,18 +47,19 @@ import { TEST_CUSTOM_DIMENSIONS } from '../../utils/data.js'
 const AREA_DIMENSION = TEST_CUSTOM_DIMENSIONS.find((dim) => dim.name === 'Area')
 
 describe('Options - Column totals', () => {
-    describe('Regression test for DHIS2-17297', () => {
-        it('does not crash', () => {
-            goToStartPage()
-            changeVisType(visTypeDisplayNames[VIS_TYPE_PIVOT_TABLE])
+    beforeEach(() => {
```

Comment:
Could we give the tests proper names instead of "Regression test...". That title was only relevant the one time. Now the name should reflect the feature that is being tested.